actors/util/adt/map.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	cid "github.com/ipfs/go-cid"
 	hamt "github.com/ipfs/go-hamt-ipld"
+	"github.com/minio/sha256-simd"
```

Comment:
Hmm, I guess we have little choice but to depend on a concrete implementation. Another thing for the vendor-for-upgrades list 😢 

Note that our use of blake2b is abstracted through a runtime syscall, but that won't be practical here.

Reply:
So, we _could_ use blake2b, but I assume that would be too slow.